content/docs/command-reference/dag.md

```diff
@@ -1,49 +1,50 @@
 # dag
 
-Show [stages](/doc/command-reference/run) in a pipeline that lead to the
-specified stage. By default it lists
-[DVC-files](/doc/user-guide/dvc-files-and-directories).
+Visualize the pipeline(s) in
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvclock-file) as a graph
```

Comment:
should we use `graph(s)`?

Reply:
The output is a single graph right? Depending what we mean by graph. One output "graph" (as in "figure") but it can have several DAGs (thus "pipeline(s)"). I think it's fine...

Reply:
Maybe change to figure actually, to avoid confusion. Or use the term DAG. See... This is why having synonyms is problematic 😢

Reply:
Updated to "DAGs" (linked to Wikipedia) in 78519465

Reply:
`visualize as graph` definitely means create a picture with graph (math term) - thus it should graph(s) since each pipeline is an independent graph.

Reply:
if we use DAGs here - let's use graph(s) (DAGs) ... DAG is too specific to be used as a main term.

Reply:
OK done!